src/Components/Member/Activities/index.jsx

```diff
@@ -1,7 +1,43 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import styles from './activities.module.css';
+import { getActivities } from 'Redux/activities/thunks';
+import { useSelector, useDispatch } from 'react-redux';
 
 const Activities = () => {
-  return <h1>Activities</h1>;
+  const dispatch = useDispatch();
+  const activities = useSelector((state) => state.activities.data.data);
+
+  useEffect(() => {
+    dispatch(getActivities());
+  }, []);
+  console.log('activities: ', activities);
```

Comment:
We should not leave console logs in the code

Reply:
uuuuuf sorry for that!